apisix/core/utils.lua

```diff
@@ -80,28 +82,18 @@ function _M.split_uri(uri)
 end
 
 
-local function dns_parse(domain, resolvers)
-    resolvers = resolvers or _M.resolvers
-    local r, err = resolver:new{
-        nameservers = table.clone(resolvers),
-        retrans = 5,  -- 5 retransmissions on receive timeout
-        timeout = 2000,  -- 2 sec
```

Comment:
We should keep the options for the new lua-resty-dns-client way. It would be better if user can configure them.

Reply:
Added it in default value for keep same with old logical.